C-Plus-Plus/sort/Bead_Sort.cpp

```diff
@@ -0,0 +1,84 @@
+/*Bead sort, also called as gravity sort, is a natural sorting algorithm since this algorithm was inspired from natural phenomenons
+and was designed keeping in mind objects(or beads) falling under the influence of gravity.The bead sort operation can be compared
+to the manner in which beads slide on parallel poles, such as on an abacus. However, each pole may have a distinct number of beads.*/
```

Comment:
Make it more compact and justify the formatting. It should be easy to read and consume